File: laravel-backend/app/Models/Student.php
<?php
namespace App\Models;
use Illuminate\Database\Eloquent\Model;
class Student extends Model {
    protected $primaryKey = 'student_id';
    public $incrementing = false;
    protected $keyType = 'string';
    protected $fillable = ['student_id','full_name','status'];
    public function assets() { return $this->hasMany(Asset::class, 'owner_student_id', 'student_id'); }
    public function exits() { return $this->hasMany(ExitLog::class, 'student_id', 'student_id'); }
}

File: backend/models/student.py
from backend.app import db
from sqlalchemy.orm import relationship

class Student(db.Model):
    __tablename__ = 'student'
    
    student_id = db.Column(db.String(20), primary_key=True)
    full_name = db.Column(db.String(100), nullable=False)
    status = db.Column(db.String(20), default='active')  # active, blocked
    
    # Relationships
    assets = relationship('Asset', backref='owner', lazy=True)
    exit_logs = relationship('ExitLog', backref='student', lazy=True)
    
    def to_dict(self):
        return {
            'student_id': self.student_id,
            'full_name': self.full_name,
            'status': self.status
        }
